pkg/code-generator/codegen/templates/event_loop_test_template.go

```diff
@@ -25,13 +30,22 @@ import (
 	"github.com/solo-io/solo-kit/pkg/api/v1/clients"
 	"github.com/solo-io/solo-kit/pkg/api/v1/clients/factory"
 	"github.com/solo-io/solo-kit/pkg/api/v1/clients/memory"
+
+	// Needed to run tests in GKE
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+
+	// From https://github.com/kubernetes/client-go/blob/53c7adfd0294caa142d961e1f780f74081d5b15f/examples/out-of-cluster-client-configuration/main.go#L31
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
```

Comment:
duplicate import here

Reply:
this gets all cloud providers, fwiw:
`_ "k8s.io/client-go/plugin/pkg/client/auth"`
learned through:
https://github.com/solo-io/squash/commit/5b4aae1b1619f6c2e061906c07e380ade10fa164#diff-70ee2ee3d399e05b88574a8000985f42R11

Reply:
Fixed thay